dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/utils/TaskExecutionCheckerUtils.java

```diff
@@ -75,14 +75,14 @@ public static void checkTenantExist(WorkerConfig workerConfig, TaskExecutionCont
     public static void createProcessLocalPathIfAbsent(TaskExecutionContext taskExecutionContext) throws TaskException {
         try {
             // local execute path
-            String execLocalPath = FileUtils.getProcessExecDir(
-                    taskExecutionContext.getProjectCode(),
-                    taskExecutionContext.getProcessDefineCode(),
-                    taskExecutionContext.getProcessDefineVersion(),
-                    taskExecutionContext.getProcessInstanceId(),
-                    taskExecutionContext.getTaskInstanceId());
-            taskExecutionContext.setExecutePath(execLocalPath);
-            FileUtils.createWorkDirIfAbsent(execLocalPath);
+            // String execLocalPath = FileUtils.getProcessExecDir(
+            // taskExecutionContext.getProjectCode(),
+            // taskExecutionContext.getProcessDefineCode(),
+            // taskExecutionContext.getProcessDefineVersion(),
+            // taskExecutionContext.getProcessInstanceId(),
+            // taskExecutionContext.getTaskInstanbu iceId());
+            // taskExecutionContext.setExecutePath(execLocalPath);
+            FileUtils.createWorkDirIfAbsent(taskExecutionContext.getExecutePath());
```

Comment:
If you leave this part out, where did you set the ```execLocalPath```?
please tell me where it is set, maybe I didn't notice it

Reply:
> 

I put it in `TaskExecutionContextBuilder` when setting some task related attribution, together with setting of `appInfoPath`.

Reply:
Your Aspect can only intercept submissions on yarn, right? If the task is not running on yarn, is it impossible to get the execution path?

Reply:
Yes, if it's not yarn job, aop code will not execute. However, all types of tasks will generate execution path (as well as appInfoPath) in master, so i don't get what you mean "impossible to get the execution path", would you like to explain it in detail?

Reply:
> Yes, if it's not yarn job, aop code will not execute. However, all types of tasks will generate execution path (as well as appInfoPath) in master, so i don't get what you mean "impossible to get the execution path", would you like to explain it in detail?

@Radeity sorry ,I found where you are setting the value. I have no doubts about this piece of code